vmdb/app/controllers/miq_request_controller.rb

```diff
@@ -476,9 +476,14 @@ def prov_set_default_options
     opts[:states] = PROV_STATES
     opts[:reason_text] = nil
     opts[:types] = request_types_for_dropdown
-    time_period = 30        # fetch uniq requesters from this time frame, since that's the highest time period in pull down.
-    conditions = ["created_on>=? AND created_on<=? AND type IN (?)", time_period.days.ago.utc, Time.now.utc, request_types_for_model.keys]
-    opts[:users] = MiqRequest.all_requesters(conditions)
+
+    opts[:users] = MiqRequest.where(
+      :type       => request_types_for_model.keys,
+      :created_on => 30.days.ago.utc..Time.now.utc
```

Comment:
It's a range...so this is saying, created on between those two dates...might be clearer in parens, maybe